docs/wave.md

```diff
@@ -145,6 +145,67 @@ The first repository is used to store the built container images. The second one
 The repository access keys must be provided as Seqera Platform credentials (see
 [Authenticate private repositories](#authenticate-private-repositories) above).
 
+### Mirroring containers
+
+Wave allows mirroring i.e. copying containers used by your pipeline into a container registry of your choice, so that
+containers are pulled by the pipeline from the target registry instead of the original registry.
```

Comment:
```suggestion
Wave allows mirroring, i.e., copying containers used by your pipeline into a container registry of your choice. This allows the pipeline to pull containers from the target registry rather than the original registry.
```